datalad/crawler/nodes/annex.py

```diff
@@ -616,16 +621,16 @@ def merge_branch(self, branch, target_branch=None,
           'theirs' with commit=False can be used to prepare data from that branch for
           processing by further steps in the pipeline
         commit: bool, optional
-          Either to commit when merge is complete
+          Either to commit when merge is complete or not
         one_commit_at_at_time: bool, optional
-          Either to generate
+          Either to generate or not
```

Comment:
I feel we better unity docs for all such flags to be uniform and probable imperative in most cases. @hanke what do you think? Also worth comparing to language  used in other popular toolboxes, eg numpy, nibabel, pandas

If imperative, In this case should be

Merge each commit of the source branch one at a time. Might be necessary to e.g. postprocess merged content for each state of the source branch